### DISCUSSION TITLE
Speed up simple tab rename action

### DISCUSSION THREAD

**Discussion by gvSmartEnergy** [furybee/chrome-tab-modifier#370] [Ideas]
**Describe the bug**
No bug, but design. Two entries are necessary although they can be identical. 
Namely, you need to enter a title for the rule and then the name for the tab. 

Currently I enter the rule and copy&paste to rename the tab as the rule is inherently linked to this one page. 

**Expected behavior**
Please consider one of the following approaches:
* Offer a setting that the rule name and tab name are filled at the same time, or the rule name is automatically applied upon save.
* Offer a tick box to take over the rule name as tab name. 

Thanks!

**Comment by sebastienfontaine** [comment]
Hello, 

Thank you for your suggestion. We’ll review it and consider the approaches you proposed.

We appreciate your input!
